astropy/time/formats.py

```diff
@@ -745,6 +746,52 @@ class TimeUnixTai(TimeUnix):
     epoch_scale = 'tai'
 
 
+class TimeUnixPTP(TimeUnix):
+    """
+    Counts the number of SI seconds elapsed since 1970-01-01 00:00:00 TAI.
+
+    This is basically the same as the ``unix_tai`` with an offset of 8 seconds
+    due to the different epoch (1970-01-01 00:00:00 UTC vs. TAI).
+
+    This will generally differ from Unix time by the cumulative integral number
+    of leap seconds since 1970-01-01 UTC plus the initial offset of 8 seconds.
+
+    This convention matches the definition of the Precision Time Protocol,
+    https://en.wikipedia.org/wiki/Precision_Time_Protocol and
+    also used by the White Rabbit protocol in High Energy Physics:
+    https://white-rabbit.web.cern.ch/
+
+    For a definition of the time scale, see:
```

Comment:
This is a bit confusing since one doesn't know the context of "the time scale", and this class is about a time format.  Maybe you mean "For more information about White Rabbit Based Clocks and Timestamps see https://www-acc.gsi.de/wiki/Timing/TimingSystemButisInterface#Clocks"?

Reply:
fixed